lib/report_plugins/daytoday.js

```diff
@@ -15,6 +15,7 @@ function init() {
   return daytoday;
 }
 
+var showInsulinUnitsWithCarbTreatment = false;
```

Comment:
how can `showInsulinUnitsWithCarbTreatment` get `true`? Adding a flag with default false sounds like adding dead code.

Reply:
Agreed.  My problem is that I see no value to that extra text.  But it's in the code, so maybe someone uses it.  I figured if people wanted it, they could fork and set the boolean to true.  Do you think anyone uses it?